docs/main/04-guides/01-installing-kratix/04-others.mdx

```diff
@@ -1,13 +1,16 @@
 ---
 description: Run Kratix
-slug: /main/guides/installing-kratix
-title: Multi cluster
+slug: /main/guides/installing-kratix-others
+title: Others (KinD etc)
```

Comment:
Maybe "Other Tools"?

Reply:
yeah, maybe `Other Providers`? Thinking for something that covers openshift/civo/kind/minikube